frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/feature-flag-root-page-content/feature-flag-root-section-card/feature-flag-root-section-card.component.html

```diff
@@ -2,9 +2,9 @@
   <!-- header-left -->
   <app-common-section-card-search-header
     header-left
-    [filterOptions]="['All', 'Name', 'Status', 'Context']"
-    [initialFilterOption]="'All'"
-    [initialSearchValue]=""
+    [filterOptions]="featureFlagFilterOption"
```

Comment:
these ought to be observables with async pipe, a la `[filterOptions] featureFlagFilterOption$ | async`. Then you don't need to create extra component variable just to reference the latest value.